### ISSUE TITLE
Detect-to-Retrieve, some questions about the paper

### ISSUE THREAD

**Issue by MartinRenaudin** [tensorflow/models#6875]
Hi,
I have some questions about the paper "Detect-to-Retrieve: Efficient Regional Aggregation for Image Search"
- To train the Delf features, would  it be good to use instead of a standard cross-entropy loss a center loss, or one like in the ArcFace paper (used for recognition task In order to minimize the intra class variance)?
- Would it be possible for the delf feature and the codebook to be learned at the same time during the recognition training? Like In the NetVLAD paper, the all network is trained together. Or is it better to train delf and codebook separetly.
Sorry if it the questions are not good, i am not sure to understand all quite well.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
